sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala

```diff
@@ -486,6 +486,7 @@ object FunctionRegistry {
     expression[XPathLong]("xpath_long"),
     expression[XPathShort]("xpath_short"),
     expression[XPathString]("xpath_string"),
+    expression[ToHiveString]("to_hive_string"),
```

Comment:
let's not register it and keep it internal. We should only use it in `df.show`

Reply:
Yea